File: test/data/Pathological.hs
{- | This is a doc cömment.
 - -}
module Pathological ( håskell ) where



-- this is a line comment {-

{- This is a nested comment.
 - It should be handled appropriately {---}
 -}
{- this is a second block comment -}

str :: String
str = "This is not the beginning of a doc comment {- "

-- | This is a doc comment {-
håskell :: Int
håskell = 3 -- not {-

longAssIdentifier :: String
longAssIdentifier = {- inline block comment -} "this string has a long-ass identifier"

File: test/data/pathological.rs
// Line comment
fn return_str() -> &'static str {
    r###"
    This is a raw string literal.

    #"

    // This should not be counted as a comment.
    /* nor should this!
    "###
}

fn return_char() -> char {
    '"'
} /* first line of block comment
  second line of block comment */

/* nested /* block comment */ */

/***/
